lib/ansible/modules/network/nxos/nxos_switchport.py

```diff
@@ -231,11 +198,13 @@ def interface_is_portchannel(interface, module):
         True/False based on if interface is a member of a portchannel bundle
     """
     intf_type = get_interface_type(interface)
+    interface_table = {}
```

Comment:
What is the point of both adding this and keeping `interface_table = None` in the `except` block? By the time we get to `if interface_table`, it will have been set to something else, and it isn't used elsewhere